blog(embertimes#146): add writeup on redesign call for contribs (#630)
ember-learn/ember-blog: source/2020-05-01-the-ember-times-issue-146.md

---

## [Call for contributors: Shipping the Ember website(s) redesign 🎨🐹](https://twitter.com/melaniersumner/status/1254497875062673411)

> Early last year the community came together to [discuss the future of the design strategy of official Ember sites](https://twitter.com/emberjs/status/1105157291806883840). After plenty of time and work investments from individual contributors, companies and the Ember Core team, the redesign for the [original RFC](https://emberjs.github.io/rfcs/0425-website-redesign.html) had [shipped successfully](https://twitter.com/emberjs/status/1230912205631213569) for the main site of the project: [emberjs.com](https://emberjs.com)!

Reviewer: What do you think about changing the redirect link for "discuss the future" from Twitter to the actual RFC, so that there's one less jump to do to see the discussion? I think we can also shorten the second half of the 2nd sentence.
```suggestion
Early last year, the community came together to [discuss the future of the design strategy of official Ember sites](https://github.com/emberjs/rfcs/pull/425). After plenty of time and work investments from individual contributors, companies and the Ember Core team, the redesign for the main site [emberjs.com](https://emberjs.com) [shipped successfully](https://twitter.com/emberjs/status/1230912205631213569)!
```

Author: That sounds great!

> But many other official resources within the Ember community are looking forward to get a new look & feel as well - and you can help with that! A [call for contributors](https://twitter.com/melaniersumner/status/1254497875062673411?s=21) is looking for those who have experience with JavaScript, HTML and CSS and who want to help shipping the website redesign to e.g. Ember's API Docs or the CLI Guides.

Reviewer: I thought we could provide the examples of sites that need a redesign in the first sentence.

I haven't been following the redesign plan closely. I added the Blog as the first example because I vaguely remember that being the priority right now? (Maybe?)
```suggestion
Many other official resources for the Ember community, such as the [Blog](https://blog.emberjs.com/), [API Docs](https://api.emberjs.com/), and [CLI Guides](https://cli.emberjs.com/), are looking forward to get a new look & feel as well - and you can help with that! A [call for contributors](https://twitter.com/melaniersumner/status/1254497875062673411?s=21) is looking for those who have experience with JavaScript, HTML and CSS and who want to help shipping the website redesign.
```

Author: Sounds awesome, thank you for the suggestion!

> Want to get started? Be sure to read the [official status update for the Ember Website](https://blog.emberjs.com/2020/04/26/update-ember-website.html) on the blog for more details on how to contribute. And feel free to reach out to the folks on the **#st-website** channel on the [Ember Discord](https://discordapp.com/invite/zT3asNS) for any questions. We're looking forward to your contributions!

Reviewer: ```suggestion
Want to get started? Be sure to read the [official status update](https://blog.emberjs.com/2020/04/26/update-ember-website.html) for more details on how to contribute. And feel free to reach out to the folks on the **#st-website** channel on the [Ember Discord](https://discordapp.com/invite/zT3asNS) for any questions. We're looking forward to your contributions!
```
